query-node/mappings/src/membership.ts

```diff
@@ -210,7 +210,14 @@ function convertBytesToString(b: Bytes | null): string {
     return ''
   }
 
-  return Buffer.from(b.toU8a(true)).toString()
+  const result = Buffer.from(b.toU8a(true)).toString()
+
+  // prevent utf-8 null character
+  if (result.match(/^\0$/)) {
```

Comment:
This is only matching a single null character right, but what if I pass `0x0000` ?